.github/ISSUE_TEMPLATE/question.md

```diff
@@ -6,7 +6,13 @@ labels: kind/question
 assignees: ''
 
 ---
+
+
 <!-- If you need to report a security issue please visit https://docs.dapr.io/operations/support/support-security-issues -->
+
+**Note:** If you have a general support question and are looking for a quicker response, please checkout our discord channel for answers from the community:
+https://aka.ms/dapr-discord
```

Comment:
For some repositories, they do not recommend ask questions with github issues, maybe we can change it after this PR?

/cc @aronchick @yaron2